orders/serializers.py

```diff
@@ -17,13 +18,26 @@ class Meta:
         ]
 
 
+class DriverUserSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = CustomUser
+        fields = ["first_name", "last_name"]  # + driver image
+
+    def to_representation(self, instance):
+        representation = super().to_representation(instance)
+
+        if instance.is_driver != True:
+            return None
+
+        return representation
+
+
 class DriverHistorySerializer(serializers.ModelSerializer):
```

Comment:
this wouldn't work, it shows me this error:
![Screenshot (355)](https://github.com/yomnaafifi/waddi/assets/101015570/9d7d7342-124a-48dc-9e9f-8fb943710cab)